centreon/doc/API/latest/onPremise/Realtime/Resources/Schema/ResourceHostDetail.yaml

```diff
@@ -1,427 +1,439 @@
-Monitoring.HostDetail:
```

Comment:
this was $ref'ed as property schema not as list of properties, root name here - syntax error